keyboards/bface/keymaps/default/keymap.c

```diff
@@ -15,27 +15,27 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include "bface.h"
+#include QMK_KEYBOARD_H
 #include "action_layer.h"
```

Comment:
Lines 19 and 20 aren't needed.

`action_layer.h` is included automatically via `QMK_KEYBOARD_H`, and `rgblight.h` is included if `RGBLIGHT_ENABLE = yes` in `rules.mk`.